core/services/eth/contracts/FluxAggregator_test.go

```diff
@@ -18,6 +19,81 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+func mustEVMBigInt(t *testing.T, val *big.Int) []byte {
+	ret, err := utils.EVMWordBigInt(val)
+	require.NoError(t, err, "evm BigInt serialization")
+	return ret
+}
+
+func testFluxAggregatorClient_AvailableFunds(t *testing.T) {
+	aggregatorAddress := cltest.NewAddress()
+
+	// is this correct?
+	const aggregatorRoundState = "c410579e"
+	aggregatorRoundStateSelector := eth.HexToFunctionSelector(aggregatorRoundState)
+
+	selector := make([]byte, 16)
+	copy(selector, aggregatorRoundStateSelector.Bytes())
+	expectedCallArgs := eth.CallArgs{
+		To:   aggregatorAddress,
+		Data: selector,
+	}
+
+	tests := []struct {
+		name         string
+		response     []byte
+		expectedLINK assets.Link
+	}{
+		{
+			"zero",
+			mustEVMBigInt(t, big.NewInt(0)),
+			*cltest.NewLink(t, "0"),
+		},
+		{
+			"non-zero",
+			mustEVMBigInt(t, big.NewInt(100)),
+			*cltest.NewLink(t, "100"),
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
+			ethClient := new(mocks.Client)
+
+			ethClient.On("Call", mock.Anything, "eth_call", expectedCallArgs, "latest").Return(nil).
+				Run(func(args mock.Arguments) {
+					res := args.Get(0)
+					err := res.(encoding.TextUnmarshaler).UnmarshalText(test.response)
+					require.NoError(t, err)
+				})
+
+			fa, err := contracts.NewFluxAggregator(
+				aggregatorAddress,
+				ethClient,
+				nil,
+			)
+			require.NoError(t, err)
+
+			res, err := fa.GetAvailableFunds()
+			require.NoError(t, err)
+			assert.Equal(t, test.expectedLINK, res)
+			ethClient.AssertExpectations(t)
+		})
+	}
+}
+
+func makeReturnData(roundID uint64, eligible bool, answer uint64) string {
```

Comment:
Good idea, making this available to the rest of the test suite.  Maybe rename it to make it clear which contract method's return data it generates.  Something like `makeRoundStateReturnData`?